common/src/main/java/com/microsoft/identity/common/internal/request/MsalBrokerRequestAdapter.java

```diff
@@ -53,7 +53,7 @@ public BrokerRequest brokerRequestFromAcquireTokenParameters(@NonNull final Acqu
                                 : null
                 ).prompt(parameters.getOpenIdConnectPromptParameter().name())
                 .claims(parameters.getClaimsRequestJson())
-                .forceRefresh(!TextUtils.isEmpty(parameters.getClaimsRequestJson()))
+                .forceRefresh(parameters.getForceRefresh())
```

Comment:
needs a merge from `dev` I guess , I believe these `forceRefresh` changes already went to dev

Reply:
resolve.d